plugin/storage/integration/elasticsearch_test.go

```diff
@@ -87,20 +87,29 @@ func (s *ESStorageIntegration) initializeES(t *testing.T, allTagsAsFields bool)
 		elastic.SetURL(queryURL),
 		elastic.SetSniff(false))
 	require.NoError(t, err)
+	defer rawClient.Stop()
 
 	s.client = rawClient
 	s.v8Client, err = elasticsearch8.NewClient(elasticsearch8.Config{
 		Addresses:            []string{queryURL},
 		DiscoverNodesOnStart: false,
 	})
 	require.NoError(t, err)
+	// Removed CloseIdleConnections call
 
 	s.initSpanstore(t, allTagsAsFields)
 
 	s.CleanUp = func(t *testing.T) {
 		s.esCleanUp(t)
 	}
 	s.esCleanUp(t)
+
+	// Add defer to close HTTP client connections
+	resp, err := http.Get(queryURL)
+	if err != nil {
+	    t.Fatal(err)
+	}
+	defer resp.Body.Close()
```

Comment:
**suggestion (performance):** Remove unnecessary HTTP request in initializeES

This HTTP request to queryURL seems unnecessary in the initialization function. It doesn't serve a clear purpose and may introduce unwanted network calls in the test setup. Consider removing this block and focus on proper resource management for the existing clients.